locks/oldspinlock.h

```diff
@@ -71,10 +71,14 @@ extern "C" size_t MyInterlockedExchange (size_t * oldval,
 #define _MM_PAUSE  {__asm{_emit 0xf3};__asm {_emit 0x90}}
 #include <windows.h>
 
-#elif defined(__GNUC__)
+#elif defined(__GNUC__) && (defined(__i386__) || defined(__x86_64__))
 
 #define _MM_PAUSE  { asm (".byte 0xf3; .byte 0x90" : : : "memory"); }
 
+#elif defined(__POWERPC__)
+
+#define _MM_PAUSE  { __asm__ volatile ("or r27,r27,r27"); }
```

Comment:
I was mystified by this until I read this post, which would be good to add in the docs along with an explanation: https://utcc.utoronto.ca/~cks/space/blog/tech/PowerPCInstructionOddity?showcomments.

Reply:
Can you add this and some explanation as a comment?

Reply:
Yeah, gonna do this today.

Reply:
UPD. Done.